generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,528 @@
+
+True or False: Generalized linear models actually still use a line for the model (like ordinary linear regression), they just transform the outcome variable first.
+
+[(X)] TRUE
+[( )] FALSE
+***
+<div class = "answer">
+
+True! 
+We can only use ordinary linear regression when the outcome variable is continuous and unbounded, so if you have, for example, a binary outcome, then a regression model won't work unless you first transform the outcome to be continuous and unbounded -- that's what generalized linear models do! 
+
+</div>
+***
+
+Which of the following best describes what a "link function" is?
+
+[( )] A link function expresses the relationship between predictors and an outcome, such as $b_0 + b_x * x$
+[(X)] A link function is a transformation to convert an outcome variable to a form that is appropriate for linear regression
+***
+<div class = "answer">
+
+A link function is a transformation to convert an outcome variable to a form that is appropriate for linear regression.
+($b_0 + b_x * x$ is an example of a linear model, not a link function.)
+
+The transformation that will work depends on the shape of the outcome variable to start with. 
+For binary outcomes, a logit transformation works well. 
+A generalized linear model with a logit link function is often called "logistic regression" for short.
+
+</div>
+***
+
+What is an appropriate link function to transform a binary outcome variable?
+
+[(X)] Logit
+[( )] Poisson
+[( )] Gamma
+[( )] Gaussian
+[( )] Inverse-Gaussian
+***
+<div class = "answer">
+
+A **logit** (log odds) link function is usually the best choice for a binary outcome variable. 
+
+Poisson regression works well for outcomes that are count data. 
+Gamma regression and inverse-Gaussian regression work well for outcomes that are positively skewed and bounded at 0. 
+
+Gaussian regression is actually just regular linear regression! 
+A Gaussian distribution, also called a [normal distribution](https://www.khanacademy.org/math/statistics-probability/modeling-distributions-of-data/normal-distributions-library/a/normal-distributions-review), is the [default assumption for linear regression models](https://www.theanalysisfactor.com/assumptions-of-linear-models/). 
+So it's actually no transformation at all!
+
+</div>
+***
+
+## Additional Resources
+
+There's an excellent chapter on [Logistic Regression](https://online.stat.psu.edu/stat504/lesson/6) published by Penn State.
+
+If you'd like to compare output from different kinds of statistical software, [UCLA's IDRE](https://stats.oarc.ucla.edu/other/dae/) has published data analysis examples for logistic regression in [R](https://stats.oarc.ucla.edu/r/dae/logit-regression/), [Stata](https://stats.oarc.ucla.edu/stata/dae/logistic-regression/), [SPSS](https://stats.oarc.ucla.edu/spss/dae/logit-regression/), [SAS](https://stats.oarc.ucla.edu/sas/dae/logit-regression/), and [MPlus](https://stats.oarc.ucla.edu/mplus/dae/logit-regression/). 
```

Comment:
I know its a bit clunkier, but this would provide much better context for folks using screenreaders

```suggestion
If you'd like to compare output from different kinds of statistical software, [UCLA's IDRE](https://stats.oarc.ucla.edu/other/dae/) has published data analysis examples for logistic regression in a variety of softwares: 
* [Logistic Regression in R](https://stats.oarc.ucla.edu/r/dae/logit-regression/) 
* [Logistic Regression in Stata](https://stats.oarc.ucla.edu/stata/dae/logistic-regression/) 
* [Logistic Regression in SPSS](https://stats.oarc.ucla.edu/spss/dae/logit-regression/)
* [Logistic Regression in SAS](https://stats.oarc.ucla.edu/sas/dae/logit-regression/)
* [Logistic Regression in MPlus](https://stats.oarc.ucla.edu/mplus/dae/logit-regression/). 
```